src/System.Private.CoreLib/shared/System/Globalization/DateTimeFormat.cs

```diff
@@ -0,0 +1,1211 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
This is change in the shared folder ... I assume that you will handhold the mirror to deal with it.